storage/Devices/Disk.cc

```diff
@@ -35,6 +35,18 @@ namespace storage
 
 
     Disk*
+    Disk::create(Devicegraph* devicegraph, const string& name, const Region& region)
```

Comment:
Do we have an alternative constructor for this that takes a size instead of a Region?

In this case, that Region will always start at 0. The sector size will normally be 512, so the only thing that is really variable is the size. Of course, disks with a different sector size will still need a Region to pass that sector size, but normally just the size will be sufficient.